packages/rn-tester-e2e/README.md

```diff
@@ -46,7 +46,7 @@ yarn start
 
 _Note: Building for the first time can take a while._
 
-Find the **RNTester.app** in `~/AndroidStudioProjects/{ProjectName}/app/build/intermediates/apk/debug/` and move the app to the following directory `/react-native/packages/rn-tester-e2e/apps`
+Find the **RNTester.app** in `~/react-native/packages/rn-tester/android/app/build/outputs/apk/jsc/debug` and move the app to the following directory `/react-native/packages/rn-tester-e2e/apps`
```

Comment:
What do you think about changing this sentence to e.g.:
Find the **RNTester.app** in `~/react-native/packages/rn-tester/android/app/build/outputs/apk/jsc/debug`, then change: `rn-tester` folder name to : `rn- tester-e2e` and move the app to the following directory `/react-native/packages/rn-tester-e2e/apps` ? What I miss here is the fact that the folder to which we should put the file does not exist yet and it needs to be created. Let me know what do you think.

Reply:
true, done